react/features/analytics/actions.js

```diff
@@ -0,0 +1,119 @@
+import JitsiMeetJS from '../base/lib-jitsi-meet';
+import { loadScript } from '../base/util';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * Loads the analytics scripts and inits JitsiMeetJS.analytics by setting
+ * permanent properties and setting the handlers from the loaded scripts.
+ * NOTE: Has to be used after JitsiMeetJS.init. Otherwise analytics will be
+ * null.
+ *
+ * @param {Store} store - The redux store in which the specified {@code action}
+ * is being dispatched.
+ * @returns {void}
+ */
+export function initAnalytics({ getState }) {
```

Comment:
Question: do those analytics scripts work when loading them if they are already loaded? On mobile the app runs continuously, it doesn't get a new runtime environment every time...